src/test/unit_tests/jest_tests/test_nc_bucket_cli.test.js

```diff
@@ -54,6 +54,12 @@ describe('manage nsfs cli bucket flow', () => {
             path: bucket_storage_path,
         };
 
+        const bucket_suffix_json = {
+            name: 'bucket3.json',
+            owner: account_defaults.name,
+            path: bucket_storage_path,
```

Comment:
I think you need to create a different `bucket_storage_path` as this one is:
`const bucket_storage_path = path.join(tmp_fs_path, 'root_path_manage_nsfs', 'bucket1');` and you want it with your bucket name ("bucket3.json").